prow/plugins/jira-linker/jira-linker.go

```diff
@@ -111,6 +110,20 @@ func handle(gc githubClient, log *logrus.Entry, config plugins.JiraLinker, event
 	return nil
 }
 
+// Returns if found, and if so respectively the ticket type (e.g. ENG) and the ticket ref (e.g. ENG-23)
+func extractJiraTicketDetails(title, ref string) (bool, string, string) {
```

Comment:
Make the types of all params explicit.  See brevi.link/golang-style (section "Make types of arguments explicit").

Reply:
Whoops, I've been violating that a lot to keep my function definitions short 😟